src/main_nep/fitness.cu

```diff
@@ -210,13 +207,17 @@ void Fitness::report_error(
     }
 
     fprintf(fid_nep, "ANN %d %d\n", para.num_neurons1, 0);
-    for (int m = 0; m < para.number_of_variables; ++m) {
+    for (int m = 0; m < para.number_of_variables_ann; ++m) {
```

Comment:
Here, should be `para.number_f_variables`, which is the total number of variables (ANN, descriptor, GNN).

Reply:
Of course, this is only relevant for later MD predction.